2015/proposals/agarwal-darshan.md

```diff
@@ -0,0 +1,137 @@
+# Title
```

Comment:
Replace with

~~~
# Enhance Amy, a Django application for managing workshops
~~~